tests/AccessCodeAddSaveDelete.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from '@playwright/test';
+import { LoginPage  as UILoginPage } from '../src/pages/ui';
+import { SettingsPage } from '../src/pages/ui';
+
+test('AccessCode', async ({ page }) => {
+  let loginPage = new UILoginPage(page);
+  let settingsPage = new SettingsPage(page);
+  await loginPage.goto(process.env.HOBBYFARM_UI_URL as string);
+  let homePage = await loginPage.fillCredentialsAndSubmit(process.env.HOBBYFARM_UI_HEADER_TITLE as string, process.env.HOBBYFARM_ADMIN_UI_USR as string, process.env.HOBBYFARM_ADMIN_UI_PWD as string);
+
+  await settingsPage.openAccessCodeAddSaveDelete(process.env.HOBBYFARM_ACCESS_CODE as string);
```

Comment:
I think the access code must be unique (does this test work when it's executed more than one time?)

Reply:
I think it would be best to generate this value to make it unique. I don't think having it as a test parameter (as environment variable) has a benefit